corehq/messaging/scheduling/scheduling_partitioned/models.py

```diff
@@ -262,10 +262,11 @@ def _passes_user_data_filter(self, contact):
             user_data = contact.get_user_data(self.domain)
         for key, value_or_property_name in self.memoized_schedule.user_data_filter.items():
             if key not in user_data:
```

Comment:
For anyone else reviewing this, I was initially confused, because all schema fields should be present in the `UserData` object returned by `get_user_data`.  However, if you expand the diff higher, you can see that `user_data` might instead be usercase properties, which wouldn't have missing fields defaulted to `""`.